changelogs/CHANGELOG-v0.4.0.md

```diff
@@ -0,0 +1,134 @@
+
+The v0.4.0 release is a minor release which fully transitions bpfman to be a
```

Comment:
Should we mention the rebranding, bpfd -> bpfman?

Reply:
Hrm yeah I guess I should :)